pyav-example/av.md

```diff
@@ -0,0 +1,22 @@
+PyAV Installation Test
+This project has easy tests to check if PyAV and some other packages (ml_dtypes, llvmlite, Pillow) are installed and working.
+
+How to Run
+Put the files example.py, subtest.py, and run_av.sh in one folder.
+
+Open a terminal and run:
+./run_av.sh
```

Comment:
If you want to keep this instruction/line, you need to add executable bit to the .sh file (755, instead of 644).